_development.yml

```diff
@@ -4,4 +4,4 @@ url: http://localhost:4000
 #   analytics --output=[dir]
 # and running from [dir]:
 #   serve
-data_url: http://localhost:3000
+data_url: http://localhost:3000/
```

Comment:
Was this change necessary for you in development? The `data_url` fields we use don't expect it to end with a trailing slash (you can see some examples of this below in the rest of the PR diff).